package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {},
   "peerDependencies": {
-    "hubot": "2.x"
+    "hubot": "3.x"
```

Comment:
Will this make it no longer viable for v2 Hubots?

Reply:
Yes that is correct.